static/js/src/advantage/subscribe/react/components/ModalSteps/StepTwo.tsx

```diff
@@ -216,6 +216,19 @@ function StepOne({ setStep, error, setError }: StepOneProps) {
     });
   };
 
+  const TrialNotAvailable = getIsFreeTrialEnabled() ? (
```

Comment:
```suggestion
  const TrialNotAvailable = () => getIsFreeTrialEnabled() ? (
```

Reply:
I mean yes we can't call it as a component because it's not one and I'm happy to change it to one but do we gain anything by doing this instead of just rendering the node like I did?

Reply:
We gain consistency 🙃  Why use a different pattern than everywhere else all of a sudden? The less you have to wonder "why" while reading code, the better.

Reply:
Fair points. But normally PascalCase naming convention indicates a _Component_. I'd rename this to `trialNotAvailable` as it's an _Element_. But it's me just nitpicking, really.

```jsx
const element = <h1>Hello, world</h1>;
```
> https://reactjs.org/docs/rendering-elements.html
https://reactjs.org/docs/components-and-props.html#rendering-a-component

Reply:
Dang! You're right, wrong case!